app/src/main/java/com/example/androidadvancedproject/data/DataSource.java

```diff
@@ -0,0 +1,21 @@
+package com.example.androidadvancedproject.data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+public class DataSource implements LodgeItemRepository{
+    private List<LodgeItemDTO> lodgeItemDtos;
+    @Override
+    public List<LodgeItemDTO> getItems() {
+        lodgeItemDtos = Arrays.asList(
+                new LodgeItemDTO(1,"Marisel", "Radu",new Date("25/12/2020"),"123456","Valenii de munte"),
```

Comment:
don't forget to format your code :)

also, you should pass here a timestamp (long) instead of a Date object.